plugins/notifications-backend/src/database/DatabaseNotificationsStore.test.ts

```diff
@@ -288,6 +299,164 @@ describe.each(databases.eachSupportedId())(
         expect(allUserNotificationsPageTwo.at(1)?.id).toEqual(id4);
         expect(allUserNotificationsPageTwo.at(2)?.id).toEqual(id3);
       });
+
+      it('should sort result', async () => {
+        const now = Date.now();
+
+        await insertNotification({
```

Comment:
You're creating a suite of tests. All these should be wrapped in a `describe` call and the setup should be moved to the `beforeAll`. Then the tests can be very short and targeted. It's odd to test that sorting works, but if it can fail then I suppose it makes sense.

Reply:
I will try to find common parts and reduce the test initialization.

These tests come from the "dependency" #23351, I will fix there. But I got your point, thanks.

Reply:
And yes, the sorting part was failing - it varies among DB engines and knex does not handle that for us.

Reply:
@planeiii-te I have updated https://github.com/backstage/backstage/pull/23351 , can you please have a look there?